test/app/features/splash/splash_page_test.dart

```diff
@@ -0,0 +1,232 @@
+          },
+        );
+
+        testWidgets(
+          'for unmapped error and in anonymous mode forward to `anonymousModeEnabled`',
+          (tester) async {
+            when(() => AppModulesMock.appConfiguration.authorizationStatus)
+                .thenAnswer((i) async => AuthorizationStatus.authenticated);
+
+            when(() => AppModulesMock.appStateUseCase.check())
+                .thenFailure((_) => ServerFailure());
+
+            when(() => AppModulesMock.userProfileStore.retrieve()).thenAnswer(
+              (_) async => FakeUserProfileEntity.anonymous(),
+            );
+
+            when(() =>
+                    AppModulesMock.modularNavigator.pushReplacementNamed(any()))
+                .thenAnswer((i) => Future.value());
+
+            await theAppIsRunning(tester, const SplashPage());
+            verify(() => AppModulesMock.modularNavigator
+                    .pushReplacementNamed('/authentication/sign_in_stealth'))
+                .called(1);
+          },
+        );
+
+        testWidgets(
+          'for unmapped error and not in security mode forward to `feed`',
+          (tester) async {
+            when(() => AppModulesMock.appConfiguration.authorizationStatus)
+                .thenAnswer((i) async => AuthorizationStatus.authenticated);
+
+            when(() => AppModulesMock.appStateUseCase.check())
+                .thenFailure((_) => ServerFailure());
+
+            when(() => AppModulesMock.userProfileStore.retrieve()).thenAnswer(
+              (_) async => FakeUserProfileEntity(),
+            );
+
+            when(() => AppModulesMock.modularNavigator
+                    .popAndPushNamed(any(), arguments: any(named: 'arguments')))
+                .thenAnswer((i) => Future.value());
+
+            await theAppIsRunning(tester, const SplashPage());
+            verify(() => AppModulesMock.modularNavigator
+                    .popAndPushNamed('/mainboard', arguments: {'page': 'feed'}))
+                .called(1);
+          },
+        );
+
+        testWidgets(
+          'goes to a quiz session for a pending quiz',
+          (tester) async {
+            when(() => AppModulesMock.appConfiguration.authorizationStatus)
+                .thenAnswer((i) async => AuthorizationStatus.authenticated);
+
+            when(() => AppModulesMock.appStateUseCase.check())
+                .thenSuccess((_) => FakeAppStateEntity.quiz());
+
+            when(() => AppModulesMock.modularNavigator
+                    .popAndPushNamed(any(), arguments: any(named: 'arguments')))
+                .thenAnswer((i) => Future.value());
+
+            await theAppIsRunning(tester, const SplashPage());
+            verify(() => AppModulesMock.modularNavigator.popAndPushNamed(
+                '/quiz',
+                arguments: any(named: 'arguments'))).called(1);
+          },
+        );
+        testWidgets(
+          'and not in security mode forward to `feed`',
```

Comment:
Faltou algo no início da descrição deste teste?

Reply:
A descrição é uma composição da hierarquia do teste, hierarquia esta definida pelos `group` e `test`. Neste caso quando o teste é executado, aparecera a mensagem 
**SplashPage in authenticated states and not in security mode forward to `feed`**